test/common/network/udp_listener_impl_test.cc

```diff
@@ -172,6 +172,82 @@ TEST_P(UdpListenerImplTest, UseActualDstUdp) {
   dispatcher_->run(Event::Dispatcher::RunType::Block);
 }
 
+// Test a large datagram that gets dropped using recvmmsg if supported.
+TEST_P(UdpListenerImplTest, LargeDatagramRecvmmsg) {
```

Comment:
I think 1.13 does not support recv mmsg.  these two tests cover the same behavior.

Reply:
Thanks! Removed recv mmsg test.